### PR TITLE
Replace logos

### PR DESCRIPTION
**Pull request by Awointa**
Replaced outdated company logos with the new branding assets.
Updated assets in the following locations:
- favicon in the metadata
- Logo component under svgs

### CODE REVIEW

**Comment by stephanniegb** [comment]
Hey! thanks for your contribution

**Comment by stephanniegb** [comment]
There are some left however:
- The favicon was not replaced.
Screenshot 2024-12-14 at 7 41 52 AM

- The logo on the create campaign page does not have a god contrast with the bg color, there are other logos in the file that you can choose from.

Screenshot 2024-12-14 at 7 42 06 AM

- the logo in the minting campaign loader is still the old logo.

Screenshot 2024-12-14 at 7 44 44 AM

File: package.json [hunk 1]
```diff
@@ -9,12 +9,12 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@cartridge/connector": "^0.5.0",
     "@starknet-react/chains": "^3.1.0",
     "@starknet-react/core": "^3.6.0",
-    "@cartridge/connector": "^0.5.0",
     "next": "^14.2.18",
```
Comment (stephanniegb) [inline package.json:15]: Please do not change the packages.

**Comment by Awointa** [comment]
thanks for the corrections, I have implemented them

File: package.json (hunk 1 above)
Comment (Awointa) [reply package.json]: thank you so much

**Comment by stephanniegb** [comment]
okay

> thanks for the corrections, I have implemented them

Confirm that you have pushed the changes cause i can see it in the reviewed Page

**Review by Awointa — COMMENTED** [review]
reviewed

**Review by Awointa — COMMENTED** [review]
thanks for the feed back, i've reviewed changes.

**Comment by stephanniegb** [comment]
I think you don't seem too understand the task, so I'll explain again
- I want you to replace the logos with the new ones; taking into consideration the contrast with the background of where you place the logo. check out the different variations of the logo in the drive
-Also want you to update the  favicon to the new logo

**Comment by stephanniegb** [comment]
Thank you for your contribution!✨

**Pull request merged**